INSTALL

```diff
@@ -402,6 +402,23 @@
   no-pic
                    Don't build with support for Position Independent Code.
 
+  no-pinso         By default OpenSSL will attempt to stay in memory until the
```

Comment:
The abbreviation `no-pinso` is rather mysterious and not everybody will know right away that it stands for "don't pin the shared library".